### DISCUSSION TITLE
Possible to trigger an automation when team wins or loses?

### DISCUSSION THREAD

**Discussion by pdawg17** [vasqued2/ha-teamtracker#34] [Ideas]
For example, I have some colored bulbs and would love to be able to make them flash/change colors when a game is over. Or when my football team scores. Is this possible? I'm loving this integration!

**Comment by Wheresitat** [comment]
Good call re winning. What attribute would that reference? I can trigger a goal i.e "When team score from entity goes from 0 - 1" but i can't seem to see an attribute for "Win"?

**Reply by pdawg17** [reply]
I'm still kind of a noob with this stuff. Can you guide me on how to trigger a goal/points?

EDIT: Aah there are attributes. I'll take a look.

**Comment by vasqued2** [comment]
Here are links to examples of automations.  Feel free to add ones that work for you to the Wiki to help others.  If you have trouble after looking at these, just ask.

From the Wiki...
[https://github.com/vasqued2/ha-teamtracker/wiki/Automation-Examples](https://github.com/vasqued2/ha-teamtracker/wiki/Automation-Examples)

From the thread in the Home Assistant forums...
[https://community.home-assistant.io/t/real-time-sports-scores-w-teamtracker-and-teamtracker-card-beta/456923/120](https://community.home-assistant.io/t/real-time-sports-scores-w-teamtracker-and-teamtracker-card-beta/456923/120)

**Comment by ikolsteren** [comment]
I as thinking something like this...(although not working when i add the game state)
`    leafs_win_lose:
      friendly_name: Leafs Game Result
      value_template: >-
        {% if states('sensor.leafs_tracker') == "POST" %}
        {% if state_attr('sensor.leafs_tracker','team_score') >= state_attr('sensor.leafs_tracker','opponent_score') %}
        Leafs Win
        {% else %}
        Leafs Lose
        {% endif %}`
Its a syntax problem and working on it now

**Reply by ikolsteren** [reply]
Might have got it

{% if states('sensor.leafs_tracker') == "POST" %}
  {% if state_attr('sensor.leafs_tracker','team_score') > state_attr('sensor.leafs_tracker','opponent_score') %}
     Leafs Win
  {% else %}
     Leafs Lose
{% endif %}
{% endif %}`

**Reply by ikolsteren** [reply]
Nope - template sensor looks ok but my automation i made didn't fire.   I was triggering the automation 
(I used the Netherlands football game as a test)
This is the automation trigger:

platform: state
entity_id:
  - sensor.netherlands_tracker
to: POST

**Comment by ikolsteren** [comment]
Going to try this for Argentina game

alias: Dutch Test
description: ""
trigger:
  - platform: state
    entity_id:
      - sensor.arg_tracker
    to: POST
condition: []
action:
  - choose:
      - conditions:
          - condition: template
            value_template: >-
              {% if state_attr('sensor.arg_tracker','team_score') >=
              state_attr('sensor.arg_tracker','opponent_score') %}

              {% endif %}
        sequence:
          - service: select.select_option
            data:
              option: Dutch Win
            target:
              entity_id: select.wled_preset
      - conditions:
          - condition: template
            value_template: >-
              {% if state_attr('sensor.arg_tracker','team_score') <=
              state_attr('sensor.arg_tracker','opponent_score') %}

              {% endif %}
        sequence:
          - service: light.turn_off
            data: {}
            target:
              entity_id: light.wled
mode: single

**Reply by pdawg17** [reply]
Thanks for working on this. It's still a little over my head (I'm getting better) so I appreciate your efforts.

**Reply by ikolsteren** [reply]
So am I. - hoping a few of us with heads together.   Good thing there are a few games on

**Reply by ikolsteren** [reply]
Automation triggered but I have an issue with my templates and conditions.
Will try later and test with the leaf game

**Comment by ikolsteren** [comment]
Ok - I seem to have got all in the one automation vs making a separate template sensor.
This works for hockey as there are no ties in the end
I wanted to update the Wiki but I cant seem to find the edit button to do so


alias: End of Game
description: ""
trigger:
  - platform: state
    entity_id:
      - sensor.team_tracker
    to: POST
condition: []
action:
  - if:
      - condition: template
        value_template: >-
          {{ state_attr('sensor.team_tracker','team_score') >
          state_attr('sensor.team_tracker','opponent_score') }}
    then:
      - your-winning-action-here
    else:
      - your-losing-action-here
mode: single
